test/edu-benefits/pages/directDeposit.unit.spec.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig1990n from '../../../src/js/edu-benefits/1990n/config/form';
+import formConfig1990e from '../../../src/js/edu-benefits/1990e/config/form';
+import formConfig5490 from '../../../src/js/edu-benefits/5490/config/form';
+
+const pageTests = (page, isRequired = false) => {
+  const { schema, uiSchema } = page;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input').length).to.equal(4);
+  });
+
+  if (isRequired) {
```

Comment:
Maybe it would make sense to make this the number of required fields, and default it to 0 (or whatever the default count is). And then instead of conditionally adding a test, just test that the right number of required fields are flagged. I think that would catch the case you're thinking about.

I don't think we should check the required property, because the goal of the test is to catch any inadvertent changes to the required fields in the form config. Unintentional changes might correctly set the required property and still be wrong.